docs/ROADMAP.md

```diff
@@ -1,53 +1,89 @@
 # kpt roadmap for 2021
```

Comment:
Please also have this as part of the FAQ:

`What's the status and roadmap of kpt?`

Reply:
done